extensions/blocks/subscriptions/shared.scss

```diff
@@ -0,0 +1,69 @@
+.wp-block-jetpack-subscriptions {
+	position: relative;
+
+	.wp-block-jetpack-subscriptions__form,
+	form {
+		display: flex;
+		align-items: start;
+
+		.wp-block-jetpack-subscriptions__textfield .components-text-control__input,
+		.wp-block-jetpack-subscriptions__button,
+		input[type="email"],
+		button {
+			line-height: initial;
+		}
+
+		.wp-block-jetpack-subscriptions__button,
+		button {
+			border-style: solid;
+			border-color: transparent;
+		}
+
+		.wp-block-jetpack-subscriptions__textfield, p#subscribe-email {
+			flex-grow: 1;
+			background: transparent !important;
+
+			.components-base-control__field,
+			.components-text-control__input,
+			input[type="email"] {
+				width: 100%;
+				margin: 0;
+			}
+		}
+
+		p#subscribe-submit,
+		p#subscribe-email {
```

Comment:
These IDs might end up in DOM twice with two blocks and should maybe be classes instead?

Reply:
This is straight from the shortcode output via the widget. I don't really want to change this because I'm sure it will result in a bunch of breakage.